test/article.test.js

```diff
@@ -61,168 +102,223 @@ describe('/articles Route', () => {
+                })
     })
 
     it('should be in sorted order from greatest to least', (done) => {
       chai.request(app)
-        .get('/articles/top')
-        .end((err, res) => {
-          checkVotesSorted(res)
-          done()
-        })
+                .get('/articles/top')
+                .end((err, res) => {
+                  checkVotesSorted(res)
+                  done()
+                })
     })
   })
 
   describe('GET /articles/:id', () => {
     it('should respond with single article of same id', (done) => {
       const id = '5846399f9ba87af2501fb035'
       chai.request(app)
-        .get(`/articles/${id}`)
-        .end((err, res) => {
-          checkSingleArticleProperties(res.body)
-          res.body._id.should.equal(id)
-          done()
-        })
+                .get(`/articles/${id}`)
+                .end((err, res) => {
+                  checkSingleArticleProperties(res.body)
+                  res.body._id.should.equal(id)
+                  done()
+                })
     })
 
     it('should respond with an error if an article id is not found', (done) => {
       const fakeId = '5846399f9ba87af2501fb03f'
       chai.request(app)
-        .get(`/articles/${fakeId}`)
-        .end((err, res) => {
-          checkValidError(res)
-          res.body.givens.id.should.equal(fakeId)
-          done()
-        })
+                .get(`/articles/${fakeId}`)
+                .end((err, res) => {
+                  checkValidError(res)
+                  res.body.givens.id.should.equal(fakeId)
+                  done()
+                })
     })
 
     it('should respond with an error if the id supplied is not in the valid id format', (done) => {
       const invalidId = 'notavalidId'
       chai.request(app)
-        .get(`/articles/${invalidId}`)
-        .end((err, res) => {
-          checkValidError(res)
-          res.body.givens.id.should.equal(invalidId)
-          done()
-        })
+                .get(`/articles/${invalidId}`)
+                .end((err, res) => {
+                  checkValidError(res)
+                  res.body.givens.id.should.equal(invalidId)
+                  done()
+                })
     })
   })
 
   describe('GET /articles/top with querys', () => {
     it('should have right format response and right properties in articles', (done) => {
       chai.request(app)
-        .get('/articles/top')
-        .query({ lastVote, lastDate })
-        .end((err, res) => {
-          checkResponse(res)
-          checkArticleProperties(res)
-          done()
-        })
+                .get('/articles/top')
+                .query({ lastVote, lastDate })
+                .end((err, res) => {
+                  checkReponse(res)
+                  checkArticleProperties(res)
+                  done()
+                })
     })
 
     it('should be in sorted order from greatest to least', (done) => {
+      const lastDate = new Date()
+      lastDate.setDate(lastDate.getDate() - 1)
```

Comment:
I dont think I should change these. The other `lastDate` is a string, this one is a different type, just the same name.